Dockerfile

```diff
@@ -34,7 +34,8 @@ RUN apt-get update && \
     /var/lib/apt/lists/* \
     /var/tmp/*
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
-  locale-gen
+    dpkg-reconfigure --frontend=noninteractive locales && \
```

Comment:
What's the change here do? Just curious

Reply:
It basically does the same as locale-gen, but its more general. Locale-gen sometimes does not work correctly in some distributions, like debian, or might not behave as intended in newer versions of ubuntu, so i just replaced it with a generalized command. We could still use locale-gen, it might even be better, just wanted to be prepared for the future, when it might not work correctly anymore.

Reply:
If it builds and works then I'm cool with the change :)